addons/purchase/models/account_invoice.py

```diff
@@ -15,6 +15,7 @@ class AccountMove(models.Model):
         states={'draft': [('readonly', False)]},
         string='Purchase Order',
         help="Auto-complete from a past purchase order.")
+    purchase_order_count = fields.Integer(compute="_compute_origin_po_count", string='Purchase Order Count', copy=False)
```

Comment:
The `copy=False` is not needed on a compute field like that (same for `sale_order_count`) but that's the last thing.
After that, it's ready to be merged I think